scripts/cnn_variant_wdl/README.md

```diff
@@ -0,0 +1,68 @@
+# gatk4-cnn-variant-filter
+
+### Purpose :
+This repo provides workflows that takes advantage of GATKs CNN tool which is a deep learning 
+approach to filter variants based on Convolutional Neural Networks. 
+
+Please read the following discussion to learn more about the CNN tool: [Deep Learning in GATK4](https://gatkforums.broadinstitute.org/gatk/discussion/10996/deep-learning-in-gatk4).
+
+### cram2filtered.wdl
+This workflow takes an input CRAM/BAM to call variants with HaplotypeCaller
+then filters the calls with the CNNVariant neural net tool using the filtering model specified.
```

Comment:
CNNVariant -> CNNScoreVariants
We have a default model based on Illumina exomes and genomes if none is specified